argo-cd-apps/base/member/infra-deployments/kueue/kueue.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: kueue
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: components/kueue
+                environment: staging
+                # This app must have cluster specific configurations.
+                # empty-base is an overlay that just has the namespace resource.
+                # This allows to gradually deploy the new UI to all the clusters.
+                clusterDir: empty-base
```

Comment:
Do we need this? The comment refers to another component, and we never actually refer to an empty-base directory in the component manifests.